### ISSUE TITLE
White mode for RGBW devices

### ISSUE THREAD

**Issue by MalteMatthey** [albertogeniola/meross-homeassistant#471]
Switching a device to white doesn't work correctly. This issue has already been raised twice in the last two years (Issue #140 & Issue #305) but both were marked as stale.

In #305 someone resolved how to fix it for scenes by manually editing the scenes.yaml file. It works but can be a lot of work if one has to manage many devices and scenes.
There are also other problems correlated with that. The device control and cards in dashboard always show the last color although the device is set to white.

I attached a video where you can see that the `color_mode `seems to remain in `hs `mode not in `color_temp` mode. It would be superb if this could be fixed. Great project btw!

https://github.com/albertogeniola/meross-homeassistant/assets/116491333/b8651dc2-0b4f-44e9-a2a0-c7271ee27846

**Comment by wnlowe** [comment]
I agree, and the solution in #305 worked for me so that each time it turns back on it does not lose it's color temp, but if I switch between HS and Color temp, it will not return to color temp from HS.
